__tests__/integration/test-config.js

```diff
@@ -0,0 +1,494 @@
+            <div class="modal-footer"><button type="button" class="btn btn-primary" data-dismiss="modal">OK</button></div></div></div></div>');
+            $nohits.modal('show');
+          }
+        } else {
+          bfestore.store = bfestore.jsonldcompacted2store(data, function(expanded) {
+            bfestore.store = [];
+            tempstore = bfestore.jsonld2store(expanded);
+            callback(loadtemplates);
+          });
+        }
+        //bfestore.n32store(data, url, tempstore, callback);
+      },
+      error: function(XMLHttpRequest, textStatus, errorThrown) {
+        bfelog.addMsg(new Error(), "ERROR", "FAILED to load external source: " + url);
+        bfelog.addMsg(new Error(), "ERROR", "Request status: " + textStatus + "; Error msg: " + errorThrown);
+      }
+    });
+  }
+
+  function retrieveLDS(uri, bfestore, loadtemplates, bfelog, callback){
+
+    var url = config.url + "/profile-edit/server/retrieveLDS";
+
+    $.ajax({
+      dataType: "json",
+      type: "GET",
+      async: false,
+      data: { uri: uri},
+      url: url,
+      success: function (data) {
+        bfelog.addMsg(new Error(), "INFO", "Fetched external source baseURI" + url);
+        bfelog.addMsg(new Error(), "DEBUG", "Source data", data);
+        bfestore.store = bfestore.jsonldcompacted2store(data, function(expanded) {
+          bfestore.store = [];
+          tempstore = bfestore.jsonld2store(expanded);
+          bfestore.storeDedup();
+          callback(loadtemplates);
+        });
+        //bfestore.n32store(data, url, tempstore, callback);
+      },
+      error: function(XMLHttpRequest, textStatus, errorThrown) {
+        bfelog.addMsg(new Error(), "ERROR", "FAILED to load external source: " + url);
+        bfelog.addMsg(new Error(), "ERROR", "Request status: " + textStatus + "; Error msg: " + errorThrown);
+      }
+    });
+  }
+
+
+  function deleteId(id, csrf, bfelog){
+    var url = config.url + "/verso/api/bfs/" + id;
+
+    //$.ajaxSetup({
+    //    beforeSend: function(xhr, settings){getCSRF(xhr, settings, csrf);}
+    //});
+
+    $.ajax({
+      type: "DELETE",
+      url: url,
+      dataType: "json",
+      csrf: csrf,
+      success: function (data) {
+        bfelog.addMsg(new Error(), "INFO", "Deleted " + id);
+      },
+      error: function(XMLHttpRequest, textStatus, errorThrown) {
+        bfelog.addMsg(new Error(), "ERROR", "FAILED to delete: " + url);
+        bfelog.addMsg(new Error(), "ERROR", "Request status: " + textStatus + "; Error msg: " + errorThrown);
+      }
+    });
+
+  }
+
+  /* Config object profiles
+   * Editor profiles are read from a WS endpoint
+   * The data are expected to be in a JSON array, with each object
+   * in the array containing a "json" property that has the profile
+   * itself. The "versoURL" variable is a convenience for setting the
+   * base URL of verso in the "config" definition below.
+   */
+  var rectoBase = "http://bibframe.org/bibliomata";
```

Comment:
Are we ok with the editor looking up the Instance json from bibframe.org for this test? E.g.

http://bibframe.org/bibliomata/profile-edit/server/whichrt?uri=http%3A%2F%2Fid.loc.gov%2Fontologies%2Fbibframe%2FInstance.json

I cannot replicate the steps of this test manually in my browser when running it locally because of the reference to that json lookup. Maybe the intention is to always look up this json from LOC?

Reply:
my experience was that this was addressed by hijacking the url for config.profiles.  I did the minimum to get the tests to pass, and there were some 404 errors noted in the console.

Reply:
Ok, I see that the `whichrt` things are shortcut if I use the `index.html` and `test-config.js` in place of the real files. This PR LGTM 👍 .